.github/workflows/merge-python-requirements-upgrade-prs.yml

```diff
@@ -63,35 +84,29 @@ jobs:
         run: |
           cd tubular
           pip install -e .
-
```

Comment:
Any particular reason for the removal of blank lines? Because these were added as per the [best practices document](https://openedx.atlassian.net/wiki/spaces/AC/pages/3548282937/GitHub+Actions+best+practices).